app/models/email_frequency.rb

```diff
@@ -0,0 +1,67 @@
+# Value Object representing the frequency with which email is sent. Used to
+#   compare two different frequencies, e.g. between a set value and calculated
+#   value.
+class EmailFrequency
+  include Comparable
+
+  # More frequent has a higher index, less frequent a lower index. The index is
+  #   used to compare the frequencies.
+  FREQUENCIES = [
+    WAIT         = "wait",
```

Comment:
maybe leave a comment explaining what this particular one means, not immediately obvious.